share/functions/vi_increment_decrement.fish

```diff
@@ -0,0 +1,34 @@
+alias vi_dec 'vi_increment_decrement subtract'
+alias vi_inc 'vi_increment_decrement add'
+
+function vi_increment_decrement --description 'increment or decrement numbers with ctrl+x ctrl+a'
+    set lnum (commandline --cursor)
+    set lstr (commandline -b)
+    set num 0
+    set split_str (string split '' $lstr)
+    set the_num (math $lnum + 1)
+    set lchar $split_str[$the_num]
+    set new_string ""
+
+    if string match -qr '^[0-9]+$' $lchar
+        if [ $argv = 'add' ]
```

Comment:
Is there a chance we can just use a `bool` or something here? A string seems wrong on many levels, but I am not very familiar with what we can work with.

Reply:
However, this breaks if you pass more or less than one argument. That should be checked above.

Use `if test "$argv[1]" = add` or `if contains -- add $argv`.